packages/common/src/ui/components/modals/AlertModal/AlertModal.tsx

```diff
@@ -1,44 +1,38 @@
-import React from 'react';
-import { Grid, Typography } from '@mui/material';
-import { DialogButton } from '../../buttons';
-import { AlertIcon } from '@common/icons';
-import { BasicModal } from '@common/components';
-
+import React, { useContext } from 'react';
+import { AlertModalContext } from './AlertModalContext';
 export interface AlertModalProps {
+  important?: boolean;
   message: React.ReactNode;
   open: boolean;
   onOk: () => void;
   title: string;
 }
 
+// This is really just a convenience component
```

Comment:
like it says.. you could just use the context and do this stuff yourself - but I wanted to prevent people from falling into the same trap that we had.

Doing it like this instantiates only one modal and allows callers to manipulate the state declaratively, which has the added benefit of meaning that I don't have to change all the current usages 😁